gips/data/landsat/landsat.py

```diff
@@ -297,39 +303,53 @@ class landsatData(Data):
                 'Z: cloud height in meters (default: 4000)'
             ],
             'nargs': '*',
-            'toa': True
+            'toa': True,
+            'startdate': date(1984, 3, 1),
+            'latency': 1,
         },
         'fmask': {
             'assets': ['DN'],
             'description': 'Fmask cloud cover',
             'nargs': '*',
-            'toa': True
+            'toa': True,
+            'startdate': date(1984, 3, 1),
+            'latency': 1,
         },
         'tcap': {
             'assets': ['DN'],
             'description': 'Tassled cap transformation',
-            'toa': True
+            'toa': True,
+            'startdate': date(1984, 3, 1),
+            'latency': 1,
         },
-        'dn': {
+        'dt': {
             'assets': ['DN'],
             'description': 'Raw digital numbers',
-            'toa': True
+            'toa': True,
+            'startdate': date(1984, 3, 1),
+            'latency': 1,
         },
         'volref': {
             'assets': ['DN'],
             'description': 'Volumetric water reflectance - valid for water only',
-            'arguments': [__toastring]
+            'arguments': [__toastring],
+            'startdate': date(1984, 3, 1),
+            'latency': 1,
         },
         'wtemp': {
             'assets': ['DN'],
             'description': 'Water temperature (atmospherically correct) - valid for water only',
             # It's not really TOA, but the product code will take care of atm correction itself
-            'toa': True
+            'toa': True,
+            'startdate': date(1984, 3, 1),
+            'latency': 1,
         },
         'bqa': {
             'assets': ['DN'],
             'description': 'LC8 band quality',
-            'toa': True
+            'toa': True,
+            'startdate': date(1984, 3, 1),
```

Comment:
This is a Landsat 8 only product and should have a start date of `date(2013, 5, 30)`.